openroad_interface/parasitic_estimation.py

```diff
@@ -0,0 +1,44 @@
+from var import directory
+import re
+import math
+
+
+def total_euclidean_distance(net_list, coord_data, unit):
+    the_component = net_list[0]
```

Comment:
Can we see if we can extract a `Half perimeter Wire Length` number from open road itself. The fact that our wirelengths are overerstimates is not ok, the HPWL should be an approximation within openroad that lines up with our estimates.

issue with this is that it uses the center of the std cell for the distances, when in reality the pins are all over, so this largely overestimates wirelength